src/oci.c

```diff
@@ -538,6 +705,11 @@ cc_oci_create_container_workload (struct cc_oci_config *config)
 	g_autofree gchar   *cwd = NULL;
 	gboolean           ret = false;
 	gchar             **args = NULL;
+	gchar             *quoted_workload_cmd = NULL;
```

Comment:
These (and other gchars in this patch) could also be "g_autofree gchar *foo = NULL" to simplify the code (no need to call `g_free()` / `g_free_if_set()`).